torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -276,6 +276,53 @@ Value* ModuleValue::asValue(const SourceRange& loc, Function& m) {
   return self_;
 }
 
+std::vector<std::shared_ptr<SugaredValue>> ModuleValue::desugarModuleContainer(
+    bool get_keys,
+    bool get_values,
+    const SourceRange& loc,
+    Function& m) {
+  // the submodules in the module list may be a mix of python objects
```

Comment:
Don't all the elements of `ModuleList/Dict/Sequential` get converted to `ScriptModule`s, so there aren't Python objects in any module containers?

Reply:
Not in the case of `ScriptModules`. This is a refactoring of the existing method.